### PR TITLE
Devs

### PR DESCRIPTION
**Pull request by Oplia512**
Fix bugs in client server tests.

### CODE REVIEW

File: src/test/java/com/java_academy/network/input/SocketInputDataProcessorTest.java [hunk 1]
```diff
@@ -77,7 +77,6 @@ private void createClientSocket(Socket clientSocket, InputDataProcessor processo
         } catch (IOException e) {
             e.printStackTrace();
         }
-
```
Comment (mhbratek) [inline src/test/java/com/java_academy/network/input/SocketInputDataProcessorTest.java:80]: :+1:

File: src/test/java/com/java_academy/network/input/SocketInputDataProcessorTest.java [hunk 2]
```diff
@@ -100,5 +99,4 @@ private void connectToClient(ServerSocket serverSocket){
             e.printStackTrace();
         }
     }
-
```
Comment (mhbratek) [inline src/test/java/com/java_academy/network/input/SocketInputDataProcessorTest.java:103]: :+1:

File: src/test/java/com/java_academy/network/output/SocketOutputDataProcessorTest.java [hunk 3]
```diff
@@ -85,12 +84,12 @@ private void createServerSocket(){
...
-    private void connectToClient(ServerSocket serverSocket){
-        try(Socket client = serverSocket.accept();
-            DataInputStream dataInputStream = new DataInputStream(client.getInputStream())){
+    private void connectToClient(ServerSocket serverSocket) {
+        try (Socket client = serverSocket.accept();
+             DataInputStream dataInputStream = new DataInputStream(client.getInputStream())) {
             String input = dataInputStream.readUTF();
             assertEquals(input, TEST_MESSAGE);
-        }catch (IOException e){
+        } catch (IOException e) {
```
Comment (mhbratek) [inline src/test/java/com/java_academy/network/output/SocketOutputDataProcessorTest.java:92]: :+1:

**Review by mhbratek — APPROVED** [review]
Good changes.

**Review by pawlusha — APPROVED** [review]

**Pull request merged**